app/models/user.rb

```diff
@@ -33,7 +33,7 @@ def email_address
     when library_id_user?
       email_from_symphony
     when webauth_user?
-      "#{webauth}@stanford.edu"
+      ldap_email
```

Comment:
what I'm missing in the PR is a test that actually exercises this change.  Is there a spec for the `email_address` method already?  Or do you need to add one?